config/_default/menus.yaml

```
@@ -50,16 +50,24 @@ main:
     weight: 1
     url: "news/"
     parent: "Community"
-  - name: "Enhancement Proposals"
+  - name: "Events"
     weight: 2
+    url: "events/"
+    parent: "Community"
+  - name: "About Hackathons"
+    weight: 3
+    url: "about-us/hackathons/"
+    parent: "Community"
```

```suggestion
  - name: "About Events"
    weight: 3
    url: "about-us/events/"
    parent: "Community"
```

It seems like this page is about all events, not just hackathons. Also I don't think this page needs to be accessible from the header/footer. Maybe just as a subpage under `events`, e.g., `events/about`?? It is a nice page though...